### DISCUSSION TITLE
Where to find the UI?

### DISCUSSION THREAD

**Discussion by canedje** [hultenvp/solis-sensor#195] [Q&A]
I installed the HACS and restartet HA.
The manual says: The integration can be configures via the UI?

I do not see a UI? Wher to find?
Thanks

**Comment by canedje** [comment]
I did find the UI.
But I can not find the plantID. It is not showing in the URL name.
Where to find?

**Reply by hultenvp** [reply]
What platform are you using? Ginlong.com or Soliscloud.com?

For Soliscloud it's in the URL. If you go to plant overview and then click on the plant then a new tab will open with plant details. The url of that page looks like https://soliscloud.com/#/station/stationdetail_1?id=xxxxxxxxxxx. Use that ID. Note that the site uses plant and station; they're the same.
For Ginlong just go to plant info tab and there you find plant ID.

**Reply by canedje** [reply]
Thanks for the clear answer.
At the moment I'm using solarman smart app.
After your advise I did login in Ginlong.com
It is accepting my login.
But I can not add my plant.
I can also not register because I do not get an verification code.
If I fill in the information form of my plant and I do next, nothing happens. The next button greys out and nothing happens. My plant will not show up
I'm confused what is going wrong
![image](https://user-images.githubusercontent.com/23150947/199926322-084991f2-d14a-4a92-9ebd-067b2c68ea03.png)

EDIT:

I was able to create a soliscloud account.
I tried to add my plant.
There is an plant created but it is not accepting my logger (I do have an old one)
I can not find an option to add my converter also?
What am I doing wrong?

**Reply by canedje** [reply]
EDIT 2:

It looks like I did succeed.
Instead of setting up a plant on my desktop I did it on my mobile.
There I was able to add my plant to Solis Home (Gilong.com)
After adding my plant I opened the Ginlong.com on my desktop and the plant was there available also.
There I was able to find mij PlantID -> Adding this to the integration in HA and the sensors are made.
They still have no values, but i quess that takes some time
Yep There is data!

Thanks for the tips

**Comment by hultenvp** [comment]
I wouldn't be surprised if this is not going to work.
Ginlong maybe works because Solis and Solarman are both ginlong brands, but I think you need to enter the Solarman url. Some users have successfully used this integration with Solarman, but I have no means to test.
A quick Google shows the url probably is https://home.solarmanpv.com/login
If you login there you should see already your pv data and also a plant is. If that is the case then use this url instead of the ginlong.

**Reply by canedje** [reply]
I did get Solis home working (Ginlong). So I'm Happy

![image](https://user-images.githubusercontent.com/23150947/200055149-4e88f562-c3d3-4842-91f2-721900ecf02b.png)

**Comment by matteoa66** [comment]
> answers

Hi I continue to not find UI, how you was able?
Tks, Matteo

**Reply by canedje** [reply]
You need to go to integrations and hit the + at the bottum and look for solis
![image](https://user-images.githubusercontent.com/23150947/201992149-e082e075-aa9a-4703-8903-356627d01170.png)

**Reply by matteoa66** [reply]
Thanks a lot!!!!!

**Reply by IAmTheNail** [reply]
Is it possible this integration is not available anymore ?

**Comment by IAmTheNail** [comment]
![image](https://github.com/hultenvp/solis-sensor/assets/66554577/3ed43aeb-2748-48e0-a1ac-0781d105724e)
